chart/templates/accounts-www/deployment.yaml

```diff
@@ -119,23 +119,23 @@ spec:
           {{- if .Values.accountsWww.livenessProbe.enabled }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.accountsWww.livenessProbe "enabled") "context" $) | nindent 12 }}
             httpGet:
-              path: /
+              path: /acc/
               port: http
           {{- else if .Values.accountsWww.customLivenessProbe }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.accountsWww.customLivenessProbe "context" $) | nindent 12 }}
           {{- end }}
           {{- if .Values.accountsWww.readinessProbe.enabled }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.accountsWww.readinessProbe "enabled") "context" $) | nindent 12 }}
             httpGet:
-              path: /health
+              path: /acc/health
```

Comment:
the health path doesn't exist in www containers, I think that we should update to `/acc/config.yaml`